### ISSUE TITLE
Expérimentation du Modèle LightGBM/XGBoost

### ISSUE THREAD

**Issue by mmassonn** [mmassonn/DPC#9]
Entraîner un modèle Gradient Boosting (LightGBM ou XGBoost) avec les données prétraitées et les nouvelles caractéristiques. Tester les hyperparamètres par défaut.
